pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -149,6 +149,10 @@ let
 
       sed -i -e '/libpci_loader.*Load/s!"\(libpci\.so\)!"${pciutils}/lib/\1!' \
         gpu/config/gpu_info_collector_linux.cc
+    '' + optionalString enableNaCl ''
+      cp -drT ${upstream-info.toolchain} native_client/toolchain
+      chmod -R u+w native_client/toolchain
+      find native_client/toolchain -type f -a -perm +0100 -exec "${./patch_toolchain.sh}" {} \;
```

Comment:
Can't we build the toolchain from source here instead of `patchelf`ing some binaries?